src/client/graphql/QuoteCart.graphql

```diff
@@ -1,3 +1,135 @@
+fragment QuoteDataFragment on BundledQuote {
+  id
+  dataCollectionId
+  currentInsurer {
+    id
+    displayName
+    switchable
+  }
+  price {
+    amount
+    currency
+  }
+  firstName
+  lastName
+  ssn
+  birthDate
+  startDate
+  expiresAt
+  email
+  typeOfContract
+  displayName(locale: $locale)
+  contractPerils(locale: $locale) {
+    title
+    description
+    covered
+    exceptions
+    info
+    icon {
+      variants {
+        light {
+          svgUrl
+        }
+      }
+    }
+  }
+  insurableLimits(locale: $locale) {
+    label
+    limit
+    description
+    type
+  }
+  insuranceTerms(locale: $locale) {
+    displayName
+    url
+    type
+  }
+  data
```

Comment:
Tested locally and Insurance selector seems not to be working. Maybe it's related with the change on `data` field. We're requesting `insuranceType` from it, right?!
 
<img width="1447" alt="Screenshot 2021-12-09 at 12 11 45" src="https://user-images.githubusercontent.com/19200662/145386530-81cb4ce1-48bb-4dfb-acfc-5da82af4dabb.png">

Reply:
> Great job!! 👑
> 
> When I try to go to the `/offer` page from `/offer-debugger` I get redirected to the landing page though 🤔 Both when I run this locally and in the new review app.

Sorry -- it needs an environment variable `ENABLE_QUOTE_CART_API=true`. I added it to the review app now!

Reply:
Aha - should that be added to `.env.example` then?

Reply:
But I've setted `ENABLE_QUOTE_CART_API=true` on my `.env` file. But I was getting `undefined` as the value for `type` query param when I try to select insurances from `InsuranceSelector`. (Can't test it again now since API is down 😢)

Reply:
@guilhermespopolin right, this is being corrected by backend now! We shouldn't need anymore changes in the PR but we need an update in Underwriter service to make it work again.